lms/djangoapps/certificates/views.py

```diff
@@ -444,6 +445,28 @@ def render_html_view(request, user_id, course_id):
     except (User.DoesNotExist, GeneratedCertificate.DoesNotExist, InvalidKeyError, CourseDoesNotExist):
         return render_to_response(invalid_template_path, context)
 
+    if 'evidence_visit' in request.GET:
```

Comment:
@Kelketek not sure I read the code right, but is it intentional that event is only emitted when assertion is *created*? If I understand right `evidence_visit` is only passed from `create_assertion` method.